composer.json

```
@@ -59,8 +59,25 @@
 		],
 		"check-complete-strict": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPress"
+		],
+		"test": [
```

```suggestion
		"check-all": [
```

Just a suggestion, but IMO `test` can easily be confused with running unit tests, while this script does much more.

---

Initially, I added `check-all`, but since I've mentioned test in the issue I was thinking it's better to leave it as in the issue. But I'm glad to see our thinking matched 👍🏼

---

The name used for the description needs to be updated too. (is the reason CI isn't passing)